packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 46.16.0 - September 26, 2019
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Retrieve Multiple Components by Metadata Type ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))
+
+#### salesforcedx-sobjects-faux-generator, salesforcedx-vscode-apex
+
+- Select SObject category for Refresh SObject Definitions command ([PR #1681](https://github.com/forcedotcom/salesforcedx-vscode/pull/1681))
+
+#### docs
+
+- Add retrieve multiple components documentation for [Org Browser](https://forcedotcom.github.io/salesforcedx-vscode/articles/user-guide/org-browser) ([PR #1667](https://github.com/forcedotcom/salesforcedx-vscode/pull/1667))
+
+- Add links at the bottom of the articles to provide feedback and log bugs ([PR #1646](https://github.com/forcedotcom/salesforcedx-vscode/pull/1646))
+
+- Add recommended timeout setting for [Prettier](https://forcedotcom.github.io/salesforcedx-vscode/articles/getting-started/prettier) setup ([PR #1647](https://github.com/forcedotcom/salesforcedx-vscode/pull/1647))
+
+- Added dropdown pre-selection setting info for use in [Apex](https://forcedotcom.github.io/salesforcedx-vscode/articles/apex/writing) ([PR #1665](https://github.com/forcedotcom/salesforcedx-vscode/pull/1665))
```

Comment:
Add dropdown list of pre-selection settings in [Apex](https://forcedotcom.github.io/salesforcedx-vscode/articles/apex/writing)

Reply:
I don't think that sounds quite right. What about 'Add pre-selection settings for Apex auto-complete dropdown list'?